dubbo-config/dubbo-config-spring/src/main/resources/META-INF/dubbo.xsd

```diff
@@ -307,6 +317,16 @@
                         <xsd:documentation><![CDATA[ The serialization protocol of service. ]]></xsd:documentation>
                     </xsd:annotation>
                 </xsd:attribute>
+                <xsd:attribute name="hessian2.request" type="xsd:string" use="optional">
+                    <xsd:annotation>
+                        <xsd:documentation><![CDATA[ request use hessian2 protocol. ]]></xsd:documentation>
+                    </xsd:annotation>
+                </xsd:attribute>
+                <xsd:attribute name="hessian.overload.method" type="xsd:string" use="optional">
+                    <xsd:annotation>
+                        <xsd:documentation><![CDATA[ hessian protocol support overload method. ]]></xsd:documentation>
+                    </xsd:annotation>
+                </xsd:attribute>

```

Comment:
Why do you put different prefixes `hessian2.request` and `hessian.overload.method`. 
Actually, i think we don't have to add xml support for every parameter, especially for parameters bind to a specific protocol, we can use `properties` or `<dubbo:parameter />` to do that. Your previous changes is ready to work, would you please revert changes adding xml, sorry for misleading you.